tests/test_symbolic_coefficients.py

```diff
@@ -112,3 +112,59 @@ def test_coefficients_w_xreplace(self):
         expected = '0.1*u(x, y) - 0.6*u(x - h_x, y) + 0.6*u(x + h_x, y) + 2'
 
         assert expected == str(eq.evaluate.lhs)
+
+    @pytest.mark.parametrize('order', [2, 4, 6])
+    def test_staggered_array(self, order):
```

Comment:
Do we wanna add a symbolic test as well that checks the obtained expression is what we want?

Reply:
Good point, I'll have a look into it